openreview/tools.py

```diff
@@ -35,10 +35,20 @@ def create_profile(client, email, first, last, middle = None, allow_duplicates =
     profile = get_profile(client, email)
 
     if not profile:
-        response = client.get_tildeusername(first, last, middle)
-        tilde_id = response['username']
 
-        if tilde_id.endswith(last + '1') or allow_duplicates:
+        try:
+            client.get_profile("~{first}_{last}1".format(first=first, last=last))
```

Comment:
We should implement the parameters, first, middle and last in the library:

https://openreview.net/user/profile?first=Melisa&last=Bok

This returns an array of profiles, if the result is empty then the profile doesn't exist

Reply:
I'm sorry I was wrong, get profile by first and last name doesn't return an exact match. I think we should use the previous validation:

```
client.gettildeusername(first , last)
```

and check the number